app/views/devise/two_factor_authentication/show.html.slim

```diff
@@ -8,7 +8,7 @@
       p
         'A one-time passcode has been sent to <strong>#{@phone_number}</strong>.
         'Please enter the code that you received.
-        'If you do not receive the code in #{otp_drift_time_in_minutes} minutes, please #{link_to 'request a new passcode', users_otp_new_path, data: { 'no-turbolink' => true }}.
+        'If you do not receive the code in #{otp_valid_for_in_words}, please #{link_to 'request a new passcode', users_otp_new_path, data: { 'no-turbolink' => true }}.
 
```

Comment:
I think that would result in a worse UX. If there is an expiration, I would want to know it beforehand, so that I know I only have x minutes to enter it, not after the fact. An analogy would be to wait until the user enters an invalid password before letting them what the password requirements are. Sites that do that make me shake my head. If there are requirements, I want to know them beforehand.

When PayPal sends you an OTP via SMS, the message tells you it expires in 5 minutes.